tests/general/test.sh

```diff
@@ -57,7 +57,7 @@ SR_PYTHON_VERSION="${SR_PYTHON_VERSION:-3.12}"
 # SR_SKIP_TAGS
 #   Ansible tags that must be skipped
 [ -n "$SKIP_TAGS" ] && export SR_SKIP_TAGS="$SKIP_TAGS"
-SR_SKIP_TAGS="--skip-tags tests::nvme,tests::infiniband"
+SR_SKIP_TAGS="--skip-tags tests::nvme,tests::infiniband,tests::bootc-e2e"
```

Comment:
Add same for tests/mssql_ha/mssql_ha.sh

Reply:
OK, done -- that doesn't have an e2e test, and this skip is meant to be temporary, but easy enough to revert.